server/events/models/models.go

```diff
@@ -41,11 +41,11 @@ type Repo struct {
 	// SanitizedCloneURL is the full HTTPS url for cloning without the username and password.
 	// ex. "https://github.com/atlantis/atlantis.git".
 	SanitizedCloneURL string
-	// Hostname of the VCS provider this repo is hosted on.
-	Hostname string
+	// VCSHost is where this repo is hosted.
+	VCSHost VCSHost
```

Comment:
Replaced the hostname with the `VCSHost` model which includes the hostname and the vcs host type.